.github/workflows/maintenance.yml

```diff
@@ -0,0 +1,139 @@
+name: Maintenance
+on:
+    workflow_dispatch:
+        inputs:
+            signing-key-id:
+                required: true
+                type: string
+                default: C8B77BCBE16CD2B94B43F9C8757397B82D4ED7B0
+
+jobs:
+    create-branch-prefix:
+        uses: ./.github/workflows/create_branch_prefix.yml
+        with:
+            prefix: 'maintenance'
+            version: $(jq -r '.version' package.json)
+            signing-key-id: ${{ inputs.signing-key-id }}
+        secrets:
+            GPG_SIGNING_KEY: ${{ secrets.GPG_SIGNING_KEY }}
+
+    prepare-maintenance:
+        name: Prepare Maintenance
+        needs:
+            - create-branch-prefix
+        runs-on: ubuntu-latest
+        outputs:
+            test-infos: ${{ steps.get-test-infos.outputs.test-infos }}
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                  ref: ${{ needs.create-branch-prefix.outputs.base-branch }}
+                  token: ${{ secrets.LASTMJS_GITHUB_TOKEN }}
+
+            - uses: ./.github/actions/setup_node
+
+            - run: npm install
+
+            # Run global prettier/lint fixes
+            - name: Run Prettier
+              run: npx prettier --write .
+
+            - name: Run ESLint
+              run: npx eslint . --fix
+
+            - uses: ./.github/actions/commit_and_push
+              with:
+                  branch-name: ${{ needs.create-branch-prefix.outputs.base-branch }}
+                  commit-message: 'Run global prettier and eslint fixes'
+                  gpg-signing-key: ${{ secrets.GPG_SIGNING_KEY }}
+                  signing-key-id: ${{ inputs.signing-key-id }}
+
+            - id: get-test-infos
+              uses: ./.github/actions/get_test_infos
+              with:
+                  directories: |
+                      ./examples
+                      ./tests
+
+    update-test-dependencies:
+        needs:
+            - prepare-maintenance
+            - create-branch-prefix
```

Comment:
Remove dependency on create-branch-prefix, do for all

Reply:
Basically there are two reasons to specify a dependency. One is to establish the run order of the jobs, the other is to get values from outputs. If you only care about the first then you only need the one dependency that run directly before it, but to get the second you need to specify all the dependencies you want the outputs from. I simplified all the ones I could, and left the rest